Source/WebCore/platform/graphics/GraphicsContextGL.h

```diff
@@ -1450,6 +1451,8 @@ class GraphicsContextGL : public RefCounted<GraphicsContextGL> {
 
     virtual void setContextVisibility(bool) = 0;
 
+    virtual bool setDrawingBufferColorSpaceIfSupported(DestinationColorSpace) = 0;
```

Comment:
If `GraphicsContextGL` is the source of truth, then this doesn't need to return `bool`.

Reply:
True, but I'll then need another function to get the maybe-currently-set colorspace. But it does sound better to have one source of truth.

Reply:
On second thought, it would mean that getting the current attribute value from JS would require a sync message (for platforms with a GPU process).
Though in this case it may be acceptable because reads should be rare.